ratis-common/pom.xml

```diff
@@ -47,7 +47,8 @@
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
-      <scope>test</scope>
+      <!-- The scope will be changed to test after all the unit test is migrated to Junit 5 -->
+      <!-- <scope>test</scope> -->
```

Comment:
@nandakumar131 , thanks a lot for updating this!  Tried to uncomment this and running subclass  tests of `BaseTest` in other modules will fail.
```
Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0:test failed:
 java.lang.NoClassDefFoundError: org/junit/jupiter/api/TestInfo: org.junit.jupiter.api.TestInfo
```

Then, the junit 5 dependencies becomes required.  Let's add the dependencies to `ratis-examples` and `ratis-test` instead of removing test scope.